gnulib/malloca.valgrind

```diff
@@ -0,0 +1,7 @@
+# Suppress a valgrind message about use of uninitialized memory in freea().
```

Comment:
This seems weird. @bertfrees Did you really change this?

Reply:
No. `git blame` says it was you.

Reply:
OK I think I understand. You deleted it in 9c7b1e8284cc8a70e90897e2ee7097b5ba563acc and I must have added it again.

Reply:
I'll make sure the file is deleted again when I merge.